src/web/client/utility/CommonUtility.ts

```diff
@@ -15,20 +15,16 @@ export function toBase64(data: string) {
     return btoa(unescape(encodeURIComponent(data)));
```

Comment:
unescape is a deprecated function and should not be used, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/unescape, my siggestion is to use a third party library for base64 encode/decode as btoa breaks for UTF-8 without unescaping, also unescpae ends up omitting some special characters

Reply:
Will take this in a later PR